testing/ef-tests/src/models.rs

```diff
@@ -308,39 +313,63 @@ pub enum ForkSpec {
     MergePush0,
     /// Cancun
     Cancun,
+    /// Cancun to Prague at time 15k
+    CancunToPragueAtTime15k,
     /// Prague
     Prague,
 }
 
 impl From<ForkSpec> for ChainSpec {
     fn from(fork_spec: ForkSpec) -> Self {
-        let spec_builder = ChainSpecBuilder::mainnet();
+        let spec_builder = ChainSpecBuilder::mainnet().clean_hardforks();
```

Comment:
This was a bug. The `spec_builder` was based on `mainnet()` but that already bakes the hardforks at the baseline, instead of starting clean.

In general, this wasn't a problem... but for Cancun, the already configured Prague timestamp interfered with a test. So a test that only expected Cancun to be the last active hardfork, suddenly found Prague activated in a further block.

The provided fix is quite clear -- it is okay to be based on mainnet, but remove all forks and let the `match` below activate exactly (and only) what is intended.